psydac/api/postprocessing.py

```diff
@@ -720,12 +724,12 @@ def __init__(self, geometry_file=None, domain=None, space_file=None, fields_file
         self._domain = domain
         self._domain_h = None
 
-        if not space_file.split('.')[-1] in ['yml', 'yaml']:
+        if not os.path.splitext(space_file)[-1] in ['.yml', '.yaml']:
             self.space_filename = space_file + '.yaml'
         else:
             self.space_filename = space_file
 
-        if fields_file.split('.')[-1] != 'h5':
+        if not os.path.splitext(fields_file)[-1] != 'h5':
```

Comment:
I think there should be no `not` here

Reply:
Fixed in [18a090c](https://github.com/pyccel/psydac/pull/234/commits/18a090c4864587c5a3f951cfdec613fedbc124d4)